src/app/sign-in/layout.tsx

```diff
@@ -1,22 +1,17 @@
 import React, { Suspense } from 'react'
 import MovieDetailSkeleton from '@/app/catalog/[id]/loading'
 import { NavBar } from '@/components/NavBar/NavBar'
-import { useSession } from '@/hooks/useSession'
-import { SessionProvider } from '@/providers/SessionProvider'
 interface LayoutProps {
   children?: React.ReactNode
 }
 
 const RootLayout: React.FC<LayoutProps> = async ({ children }) => {
-  const { hasSession, user } = await useSession()
   return (
-      <SessionProvider hasSession={hasSession} user={user}>
```

Comment:
removed the sessionProvider because it already exists on the global `layout.tsx`